rest_rules.cpp

```diff
@@ -439,6 +439,13 @@ int DeRestPluginPrivate::createRule(const ApiRequest &req, ApiResponse &rsp)
 
         if ((map["name"].type() == QVariant::String) && !name.isEmpty())
         {
+            if (name.size() > MAX_RULE_NAME_LENGTH)
+            {
+                rsp.list.append(errorToMap(ERR_MISSING_PARAMETER, QString("/rules/name"), QString("invalid/missing parameters in body")));

```

Comment:
Doesn't ERR_INVALID_VALUE fit in better here?

Reply:
Well, yes. Smanar also mentioned something similar. We should generally rework the API parts and make it more consistent and robust.

Was just going with the (erroneous) flow here, but happy to amend 😁